src/utils/clients.py

```
@@ -504,18 +547,21 @@ async def honcho_llm_call_inner(
                         model=model,
                         contents=prompt,
-                        config={
-                            "response_mime_type": "application/json"
-                            if json_mode
-                            else None,
-                        },
+                        config=config,  # pyright: ignore[reportArgumentType]
                     )
                 )
 
+                logger.info(f"DEBUG: Gemini Response: {gemini_response.text}")
```

Ruff (0.14.2)

Logging statement uses f-string

(G004)